alfresco-docker-libreoffice/Dockerfile

```diff
@@ -18,8 +18,8 @@ COPY target/alfresco-docker-libreoffice-${env.project_version}.jar libreoffice-d
 
 RUN ln /alfresco-docker-libreoffice-${env.project_version}.jar /usr/bin/alfresco-docker-libreoffice.jar && \
     yum install -y cairo cups-libs libSM && \
-    test -f libreoffice-dist-5.4.6-linux.gz && \
-    ln -s libreoffice-dist-5.4.6-linux.gz libreoffice-dist-linux.gz || \
+    test -f libreoffice-dist-6.1.6-linux.gz && \
```

Comment:
If I owned this code I would not have the version number in the code. Currently it needs to be updated in 3 places in this file and also in the executor (if the minor version changes).

Reply:
Done. I've added Dockerfile arguments for the _LibreOffice_ & _ImageMagick_ versions.